examples/tutorials/setup_poc.ipynb

```diff
@@ -34,10 +36,47 @@
    "id": "87989e99",
    "metadata": {},
    "source": [
-    "### Preparing the POC environment\n",
-    "Before running POC mode, there are a couple important environment variables that should be set.\n",
+    "> If you are the NVFLARE user prior to 2.4.0, you might noticed two things\n",
+    "> * poc command is now separate the sub-commands\n",
+    "> * the sub-commands such as \"prepare\", \"start/stop\" etc. are different from previous version where \"--\" is removed for command, such as \n",
+    ">   \"--prepare\" becomes \"prepare\" and \"--start\" becomes \"start\" etc. \n",
+    "\n",
+    "We can further look at the help description for each command\n",
+    "\n",
+    "> We will need ```tree``` command, so we will install ```tree`` in linux, if tree is not avaialble to you, you can replace it with 'ls -al`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bb926188-020e-4699-9411-d9970521fe0f",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "! python -m pip install tree"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2b0cf1fe-6ed4-4fc6-8b42-e43f0b28443c",
+   "metadata": {},
+   "source": [
+    "### Preparing the POC Configuration\n",
+    "Before running POC mode, there are a few environment variables you might consider to set. These env. variables are consider nice to have but not required since 2.4.0. We have provided other options if you preference is not use env. variables. \n",
     "\n",
-    "First, to simplify deploying the example apps in the NVFlare GitHub repo, you can set `NVFLARE_HOME` to the root of the GitHub clone.  "
+    "##### **NVFLARE_HOME**\n",
+    "\n",
+    "> Note: Set NVFLARE_HOME env. variables if only you want to use examples as jobs in POC mode.\n",
+    "\n",
+    "Prior 2.4.0 release, the NVFLARE_HOME env. variable is used to find the NVFlare/exampels location. In 2.4.0, we have added \n",
```

Comment:
Prior to the 2.4.0 release, the NVFLARE_HOME env. variable was used to find the NVFlare/examples location.